docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -44,9 +44,9 @@ This command provides node public key related actions.
 
 !!!caution
 
-    To get the public key or address of an existing node, ensure you use the
+    To get the public key or address of a node, ensure you use the
     [`--data-path`](CLI-Syntax.md#data-path) or
-    [`--node-private-key-file`](CLI-Syntax.md#node-private-key-file) option with the `public-key
+    [`--node-private-key-file`](CLI-Syntax.md#node-private-key-file) option with the `public-key`
```

Comment:
Add a link to the command on the `public-key` word?

Reply:
I didn't put a link on this because it's already inside the command.